units/emergency.service.in

```diff
@@ -13,6 +13,8 @@ Conflicts=shutdown.target
 Conflicts=rescue.service
 Conflicts=syslog.socket
 Before=shutdown.target
+Before=rescue.service
+Before=syslog.socket
```

Comment:
not following on the syslog.socket one. can you elaborate on the rationale for that one?

I mean, why precisely should we treat this unit specially, and differently from all other socket units?

And if this really is a good thing to have, why not have the dep in syslog.socket instead? (after all a system without syslog.socket is more easily thinkable than a system without emergency mode if you follow what i mean, and hence they latter should not reference the former, but vice versa)

Reply:
Well, I didn't put it there :).  History says something like it was getting repeatedly activated by journald in emergency mode.  So we repeatedly attempted to start syslog.service, even though we didn't expect sysinit.target to be reachable (because emergency)?

Ah yes, the reason syslog.socket is special is that it has `DefaultDependencies=no`, so it doesn't `Require=sysinit.target`.  I agree this point is easier to understand if we put the conflict there as well, in syslog.socket.

Reply:
new commit (2nd one) should address this